internal/v2/remote_signer/remote_signer_test.go

```diff
@@ -96,19 +97,17 @@ func TestSign(t *testing.T) {
 	if err != nil {
 		log.Fatalf("failed to parse cert: %v", err)
 	}
+	testInBytes := []byte("Test data.")
+	hsha256 := sha256.Sum256([]byte(testInBytes))
+
+	// Test RSA PKCS1v15 signature algorithm.
 	s := New(clientx509Cert, cstream, &commonpbv1.Identity {
 		IdentityOneof: &commonpbv1.Identity_Hostname {
 			Hostname: "client_hostname",
 		},
-	})
-	testInBytes := []byte("Test data.")
+	}, s2av2pb.SignatureAlgorithm_S2A_SSL_SIGN_RSA_PKCS1_SHA256)
+	var signerOpts crypto.Hash = crypto.SHA256
```

Comment:
readability nit: please replace with `signerOpts := crypto.SHA256`, or possibly even omit the local variable.